src/main/java/org/alliancegenome/curation_api/model/entities/Allele.java

```diff
@@ -212,4 +205,15 @@ public class Allele extends GenomicEntity {
 		@Index(name = "allele_note_relatednotes_index", columnList = "relatedNotes_id")})
 	private List<Note> relatedNotes;
 
+
+	@IndexedEmbedded(includePaths = {
```

Comment:
What's the reason for moving this from GenomicEntity to the child classes?  If this is necessary then shouldn't it also be on the various other GenomicEntity classes (Transcript, Exon, etc.).  Or is this to avoid joining all those tables, in which case do we really need a model change similar to the one for disease genetic modifiers?

Reply:
Yes, we did this to avoid joining all other genomic entities.

Reply:
This needs a LinkML model change then